core-developers/committing.rst

```diff
@@ -80,21 +80,20 @@ to enter the public source tree. Ask yourself the following questions:
 Updating NEWS and What's New in Python
 --------------------------------------
 
-Almost all changes made to the code base deserve an entry in ``Misc/NEWS.d``.
-If the change is particularly interesting for end users (e.g. new features,
-significant improvements, or backwards-incompatible changes), then an entry in
-the ``What's New in Python`` document (in ``Doc/whatsnew/``) should be added
-as well. Changes that affect documentation only generally do not require
-a ``NEWS`` entry.
+Changes that require NEWS entries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-There are two notable exceptions to this general principle, and they
-both relate to changes that:
+Most changes made to the code base deserve an entry in :cpy-file:`Misc/NEWS.d`,
+except for the following:
 
-* Already have a ``NEWS`` entry
-* Have not yet been included in any formal release (including alpha
-  and beta releases)
+* documentation changes in general
+* test changes in general
+* strictly internal changes with no user-visible effects
+* changes that already have a ``NEWS`` entry
+* changes that have not yet been included in any formal release
```

Comment:
Continuing the discussion from https://github.com/python/devguide/pull/1238#discussion_r1417664126.

I pretty much just reused the original wording, but I agree it is slightly confusing. It helps with the explanation a few lines below, though.

Reply:
I think "reverts" is ok, so I'm inclined to apply Hugo's suggestion. For anyone familiar with Git, it makes sense as a noun (`git commit`, `git revert`). I think Ezio's suggestion reads a little off-kilter (perhaps because of the double "being").

The idea of a bad/good list could work out, but I'd prefer if we could do that in a follow-up PR.

Thanks, both of you.

Reply:
Let's go for "revert" for now. There's always room for further improvements in future PRs :)